pkgs/development/python-modules/vapoursynth/default.nix

```diff
@@ -0,0 +1,27 @@
+{ vapoursynth, cython, buildPythonPackage }:
+buildPythonPackage {
```

Comment:
```suggestion
{ buildPythonPackage, cython, vapoursynth }:

buildPythonPackage {
```

Reply:
I was also thinking this but I thought it was too nitpicky :laughing: